integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/mock/answer/AbstractRequestHandler.java

```diff
@@ -54,7 +52,8 @@ public HttpMock getMock() {
     }
 
     public HttpMock clearDefinedResponses() {
-        mock.getDynamicRouteStack().clear(method.getHttpMethod(), path);
+//        mock.getDynamicRouteStack().clear(method.getHttpMethod(), path);
```

Comment:
This comment seems unnecessary

Reply:
I don't know yet. When I will refactor the verification part, I will decide that if need or not. I need to broke to more parts because it would be a gigantosaurus PR